docker-compose.yml

```diff
@@ -31,6 +31,14 @@ services:
       /usr/bin/mc mb --quiet local/easi-app-file-uploads/;
       /usr/bin/mc policy set public local/easi-app-file-uploads;
       "
+  lambda:
+    image: lambci/lambda:go1.x
+    ports:
+      - 9001:9001
+    environment:
+      - DOCKER_LAMBDA_STAY_OPEN=1
+    volumes:
+      - ./../easi-infra-modules/lambda/prince/build:/var/task:ro,delegated
```

Comment:
This likely won't work in CI (especially the reference to the volume). We could move the `lambda` service to docker-compose.local.yml or docker-compose.override.yml depending on when/how it's intended to be used.

Reply:
I moved it to `docker-compose.local.yml` for now, since I haven't had a chance to think about CI too much (and we don't test the PDF generation there yet).

Could host a zip file on S3 of a working lambda that we could fetch in a setup script? That should suffice for both local development and on CI.

Reply:
That might work, but we can probably iron out the testing issues under EASI-1057.